py-getting-started.Rmd

```diff
@@ -106,10 +107,35 @@ it becomes quick and easy to run code that's written in a script. Try "running"
 the lines in your script. You should see their output in the console on the 
 bottom right. 
 
+### Organizing Sections with Code Cells
```

Comment:
I think it is important to introduce this early, since it is integral to how to think about organization within scripts (imo) and analogous to R code chunks.

Reply:
this is great - I only know about cells in the context of a notebook and haven't done it from Spyder.